doc/UserGuide.md

```diff
@@ -57,7 +57,7 @@ Format: `list`
 Finds persons whose names contain any of the given keywords.<br>
 Format: `find KEYWORD [MORE_KEYWORDS]`
 
-> The search is case sensitive, the order of the keywords does not matter, only the name is searched, 
+> The search is case insensitive, the order of the keywords does not matter, only the name is searched, 
 and persons matching at least one keyword will be returned (i.e. `OR` search).
 
 Examples: 
```

Comment:
Should add an example of usage of case-insensitive find, e.g. 
`find john`
Returns `John Doe`